frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/entity/AccountInterventionsResponse.java

```diff
@@ -1,11 +1,17 @@
 package uk.gov.di.authentication.frontendapi.entity;
 
 import com.google.gson.annotations.Expose;
-import org.apache.logging.log4j.core.config.plugins.validation.constraints.Required;
```

Comment:
Was the issue here actually that this is the wrong import? Should be 
`import uk.gov.di.authentication.shared.validation.Required;`. Looks like this mistake proliferated a bit in our AIS code

Reply:
(I would be sad if we had to start eroding nice un-boilerplate-y records with constructors etc but haven't tested this out fully)